lib/menu.rb

```diff
@@ -0,0 +1,23 @@
+class Menu
+  attr_reader :items
+
+  def initialize
+    @items = {}
+  end
+
+  def add(key, item)
+    @items[key] = item
+  end
+
+  def remove(key)
```

Comment:
Since the key is not printed, in `#print_menu` is it's only use for deleting items?